app/models/chargeback/consumption.rb

```diff
@@ -10,7 +10,13 @@ def consumed_hours_in_interval
       #   2) We cannot charge for future hours (i.e. weekly report on Monday, should charge just monday)
       #   3) We cannot charge for hours after the resource has been retired.
       @consumed_hours_in_interval ||= begin
-                                        consumption_start = [@start_time, resource.try(:created_on)].compact.max
+                                        born_at = resource.try(:created_on)
+                                        if born_at && born_at > @start_time
+                                          # run this extra SELECT only when useful -- if resource was born in the current interval
+                                          # metrics can be older than resource (first capture may go few days back)
+                                          born_at = [born_at, resource.first_capture].compact.min
```

Comment:
@isimluk  can we somehow reuse `first_metric_rollup_record` what we already have (instead of `first_capture`) ?